settings/post-process.sql

```diff
@@ -95,3 +95,80 @@ FROM
 WHERE
   osm_bus_points.osm_id = osm_bus_route_members.member_osm_id
 ;
+
+-- Deduplicate linstring
+DROP TABLE IF EXISTS osm_bus_ways;
+CREATE TABLE osm_bus_ways AS
```

Comment:
We need better names ... `osm_bus_ways` and `transport_bus_ways` are not very good ones.
Suggestions 
* `osm_bus_ways` >> `ways_with_line_shields` or `deduplicated_ways`
* `transport_bus_ways` >> `duplicated_ways` or `bus_ways`

Reply:
The naming can be better.
But your suggestion ways_with_line_shields is too linked to only one possible usage.
The idea of table I introduced is really to replace the original ones. The first is already no more used. The second still temporary used.
But my main idea is that all the database layout should be rewritten.